src/config.cpp

```diff
@@ -364,7 +366,7 @@ void parse_program_options(int argc, const char** argv)
         ("metric-leader",
             po::value(&config.metric_leader)
                 ->value_name("EVENT")
-                ->default_value(QUOTE(DEFAULT_METRIC_LEADER)),
+                ->default_value(perf::EventProvider::get_default_metric_leader_event().name),
```

Comment:
On a second thought, I'm not convinced that this is a good idea. Using perf in program options arg parsing has some problems, e.g.: perf might not be available, there might not be any suitable metric leaders, it might cause some latency, which may be a bad idea for shell autocompletion, ...

Instead, remove the default_value and either use the given metric_leader, or determine a possible metric leader later on.